packages/strapi-plugin-upload/admin/src/components/DropdownSection/index.js

```diff
@@ -6,7 +6,7 @@ const DropdownSection = styled.div`
   display: none;
   position: absolute;
   top: 38px;
-  left: 0;
+  left: 38px;
```

Comment:
It seems this compo is also used in the ListView and may change the UI.
Can you please check it doesn't break the alignment?

Reply:
Very good catch @virginieky ! Thank you.